src/shared/components/unsupportedBrowserModal/UnsupportedBrowserModal.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import UnsupportedBrowserModal from "./UnsupportedBrowserModal";
+import browser from 'detect-browser';
+
+describe('UnsupportedBrowserModal', () => {
+
+    let wrapper: any;
+
+    before(()=> {
+         wrapper = shallow(<UnsupportedBrowserModal/>);
+    });
+
+    it.only('does not show modal when Chrome is being used', () => {
```

Comment:
@Tamba32 This still has `only` parameters, need to remove those (you can see in the code coverage report that everything turns red)